open-company/open-company-web
# Section editing

card: https://trello.com/c/957XqBVY

To test:
- run `lein test-all!`
- checkout tests for `test.open-company-web.finances.finances`, `test.open-company-web.growth.growth` and `test.open-company-web.simple-section`: they should cover a good part of the editing states
- check the code
- try to edit a simple section
- try to add a new section and save the edited data (this needs a fix on the API side: right now a PATCH with `:sections` and the new section doesn't return `:updated-at` and `:author`)
- try to edit a _finances_ section (it's guaranteed to update only title and notes for now)
- try to edit a _growth_ section (it updates only title and notes for now)
- merge if you like
- send a newsletter via torrent in clojurescript
